src/style.css

```diff
@@ -1 +1,102 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir-Book;
+  src: url(fonts/Avenir-Book.ttf) format('truetype');
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(fonts/Avenir-Heavy.ttf) format('truetype');
+}
+
+body {
+  font-family: Avenir-Book, sans-serif;
+  font-weight: 300;
+}
+
+.search_bar {
+  height: 68px;
+  width: 100%;
+  margin-top: 20px;
+
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  display: flex;
+  align-items: center;
+
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+}
+
+.search_bar--small {
+  height: 40px;
+}
+
+.search_bar__logo {
+  background-image: url(images/Search.svg);
+  background-size: contain;
+}
+
+.search_bar__logo--large {
+  height: 19px;
+  width: 19px;
+  margin: 0 18px 1px 25px;
+}
+
+.search_bar__logo--small {
+  height: 11px;
+  width: 11px;
+  margin: 0 10px 1px 12px;
+}
+
+.search_bar__input {
+  padding: 0;
+  height: 19px;
+  width: 158px;
+  border: none;
+  background: none;
+}
+
+.search_bar__input--small {
+  margin-bottom: 1px;
+}
+
+.search_bar__input::placeholder {
+  font-family: Avenir-Book, sans-serif;
+  font-weight: 300;
+  font-size: 16px;
+}
+
+.search_bar__input--small::placeholder {
+  font-size: 14px;
+  font-weight: 400;
+}
+
+.search_bar--large:hover {
+  box-shadow: 0 4px 4px 0 rgba(0, 0, 0, 0.25);
+}
+
+.search_bar__input:focus {
+  outline: none;
+  font-family: Avenir-Heavy, sans-serif;
```

Comment:
Remember to use fallback fonts - alternative font-family in case the main one doesn't work. Consider adding a more generic family like `serif` as a fallback for Avenir-Heavy.